src/App.js

```diff
@@ -1,11 +1,10 @@
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <h1>
-        hello react
-      </h1>
+    <div>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Please use class-based component instead function-based component as this is the major requirement of the project.

### See the image of the requirement 

![image](https://user-images.githubusercontent.com/83803414/176022694-ef386b3d-8c2a-4606-a861-de5812019264.png)